src/tn/eduVision/services/SallesService.java

```diff
@@ -0,0 +1,194 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package tn.eduVision.services;
+
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+import tn.eduVision.entités.Salle;
+import tn.eduVision.entités.TypeSalle;
+import tn.eduVision.exceptions.NoDataFoundException;
+import tn.eduVision.tools.CustomLogger;
+import tn.eduVision.tools.SqlConnectionManager;
+
+/**
+ *
+ * @author job_j
+ */
+public class SallesService implements Iservices<Salle> {
```

Comment:
also IService is better than Iservices , naming not a big deal . also for betting califying should we change methods to start with capital letters ? Add , Update....

Reply:
i'll update it fi master